pkg/operator/controllers/subnets/subnets_controller.go

```diff
@@ -142,7 +142,7 @@ func (r *reconcileManager) reconcileSubnets(ctx context.Context) error {
 	}
 
 	if len(combinedErrors) > 0 {
-		return fmt.Errorf(strings.Join(combinedErrors, "\n"))
+		return fmt.Errorf("%s", strings.Join(combinedErrors, "\n"))
```

Comment:
Instead of doing a no-op format, use

```go
return errors.New(strings.Join(combinedErrors, "\n"))
```

Or use a utility for error aggregation - the k8s libraries have some, etc.

Reply:
Addressed